app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,46 @@
+<h2 class="headline">Sign up</h2>
+
+<% stylesheet_link_tag 'application', media: 'all' %>
+
+<div class="flash-messages">
+  <% flash.each do |key, value| %>
+    <div class="alert alert-<%= key %>">
+      <%= value %>
+    </div>
+  <% end %>
+</div>
+
+<div>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name), id: "form-container") do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <% f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, class: "form-control", width: "100px", placeholder: "Name:" %>
+    </div>
+    <br />
+    <div class="field">
+      <% f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control", width: "100px", placeholder: "Email:" %>
+    </div>
+    <br />
+    <div class="field">
+      <% f.label :password %>
+      <% if @minimum_password_length %><br />
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password", class: "form-control", width: "100px", placeholder: "Password:" %>
+    </div>
+
+    <div class="field">
+      <% f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "form-control", width: "100px", placeholder: "Password confirm:" %>
+    </div>
+    <br />
+    <div class="actions">
+      <%= f.submit "Sign up", class: "submit-btn2" %>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] @Team Great work making changes in your signup/login page, but their layout still doesn't match the given design, all the input fields and buttons should be of equal width and aligned in a straight height, check the comparison below

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/jcmunav63/my-ror-blog/assets/61408860/f61c75ee-407d-420a-b0bc-c4bc303c4b1e) | ![image](https://github.com/jcmunav63/my-ror-blog/assets/61408860/1fa36dea-05e6-4d59-9a8e-6e04427046c1)